README.md

```diff
@@ -129,16 +129,16 @@ An interface to a deployed Step Function, with a function to execute a Step Func
 ### Arguments
 
 - When running tests with `jest` using `sls-test-tools` matchers there are certain parameters needed for `sls-test-tools` to make assertions.
-- These are passed as command line arguments, using quotation to match `jests` convention on test arguments.
+- These are passed either as command line arguments, using quotation to match `jests` convention on test arguments, or by using environment variables. CLI arguments override environment variables.
 
 **Required**
 
-- `'--stack=my-service-dev'` - the CloudFormation stack name of the stack under test.
+- `'--stack=my-service-dev'` or `process.env.AWS_STACK_NAME` - the CloudFormation stack name of the stack under test.
```

Comment:
I'm not sure about this variable name. `AWS_PROFILE` and `AWS_REGION` are official AWS variables. (Recognized by AWS SDK or CLI). `AWS_STACK_NAME` is custom to `sls-test-tools`. 

Moreover, the stack name is scoped to CloudFormation, not to AWS.
I propose `CFN_STACK_NAME`.

Reply:
Yes it makes more sense ! I've updated the PR accordingly